luatest/output_beautifier.lua

```diff
@@ -154,7 +163,10 @@ function OutputBeautifier.mt:run(fd, pipe)
             end
             for _, line in pairs(lines) do
                 line_color_code = self:color_for_line(line) or line_color_code
-                io.stdout:write(table.concat({prefix, line_color_code, line, self.class.RESET_TERM, '\n'}))
+                if rawget(_G, 'unified_logfile') ~= nil then
+                    rawget(_G, 'unified_logfile').fh:write(table.concat({prefix, line, '\n'}))
+                end
+                io.stdout:write(table.concat({colored_prefix, line_color_code, line, self.class.RESET_TERM, '\n'}))
```

Comment:
This code always writes the captured `stdout` and `stderr` to the `stdout` of our current `luatest` process. This is not very good. It will need to be fixed in the future.

Reply:
This has been fixed.